### PR TITLE
Enable primitive types to be inserted with BulkInsert

### LINKED ISSUE #114: Support inserting primitive types using Bulkinsert
When we want to bulk insert something, it must be an object. So if we want to insert list of numbers, we need to map it to some object with property and insert those. I would be nice to have direct insertions of such data.

### PR DESCRIPTION
**Pull request by sincoda**
Closes #114

### CODE REVIEW

File: Kros.Utils/tests/Kros.Utils.UnitTests/Data/BulkInsert/SqlServerBulkHelper.cs [hunk 1]
```diff
@@ -17,6 +17,20 @@ public static void CompareTables(DataTable actualTable, DataTable expectedTable)
             CompareData(actualTable, expectedTable);
         }
 
+        public static void CompareColumnValues(DataTable actualTable, DataTable expectedTable, string columnName)
+        {
+            actualTable.Columns[columnName].Should().NotBeNull($"Tabuľke chýba stĺpec [{columnName}].");
```
Comment (LukasSefcik) [inline Kros.Utils/tests/Kros.Utils.UnitTests/Data/BulkInsert/SqlServerBulkHelper.cs]: Should we use english language?

File: Kros.Utils/tests/Kros.Utils.UnitTests/Data/BulkInsert/SqlServerBulkHelper.cs [hunk 2]
```diff
@@ -17,6 +17,20 @@ public static void CompareTables(DataTable actualTable, DataTable expectedTable)
...
+
+            actualTable.Rows.Count.Should().Be(expectedTable.Rows.Count,
+                $"Tabuľky majú rôzny počet stĺpcov: actual = {actualTable.Rows.Count}, " +
```
Comment (LukasSefcik) [inline Kros.Utils/tests/Kros.Utils.UnitTests/Data/BulkInsert/SqlServerBulkHelper.cs]: Same here.

File: Kros.Utils/tests/Kros.Utils.UnitTests/Data/BulkInsert/SqlServerBulkHelper.cs [hunk 3]
```diff
@@ -90,12 +119,15 @@ private static void CompareRowData(DataRow actualRow, DataRow expectedRow)
...
+                if (string.IsNullOrEmpty(columnName) || columnName.ToUpper().Equals(column.ColumnName.ToUpper()))
+                {
+                    object expectedValue = expectedRow[column] == DBNull.Value ? "NULL" : expectedRow[column];
+                    object actualValue = actualRow[column.ColumnName] == DBNull.Value ? "NULL" : actualRow[column.ColumnName];
 
-                actualValue.Should().Be(expectedValue, $"Riadok s primárnym kľúčom \"{pk}\" nemá požadované dáta. " +
-                    $"V stĺpci [{column.ColumnName}] je hodnota \"{actualValue}\", " +
-                    $"očakávaná hodnota je \"{expectedValue}\".");
+                    actualValue.Should().Be(expectedValue, $"Riadok s primárnym kľúčom \"{pk}\" nemá požadované dáta. " +
+                        $"V stĺpci [{column.ColumnName}] je hodnota \"{actualValue}\", " +
```
Comment (LukasSefcik) [inline Kros.Utils/tests/Kros.Utils.UnitTests/Data/BulkInsert/SqlServerBulkHelper.cs]: Translate to english language?

File: Kros.Utils/tests/Kros.Utils.UnitTests/Data/BulkInsert/SqlServerBulkInsertShould.cs [hunk 4]
```diff
@@ -366,6 +395,18 @@ public void ThrowInvalidOperationExceptionWhenColumnDoesNotExistInDestinationTab
             },
         };
 
+        private static List<string> _rawDataShortText = new List<string>
```
Comment (LukasSefcik) [inline Kros.Utils/tests/Kros.Utils.UnitTests/Data/BulkInsert/SqlServerBulkInsertShould.cs]: Maybe just `IEnumerable<string>` is good enough.

File: Kros.Utils/tests/Kros.Utils.UnitTests/Data/BulkInsert/SqlServerBulkHelper.cs [hunk 5]
```diff
@@ -61,20 +75,35 @@ private static void CompareColumns(DataTable actualTable, DataTable expectedTabl
...
 
         private static void CompareData(DataTable actualTable, DataTable expectedTable)
+        {
+            foreach (DataRow expectedRow in expectedTable.Rows)
+            {
+                CompareRowOneColumn(actualTable, expectedTable, expectedRow);
+            }
+        }
+
+        public static void CompareRowOneColumn(DataTable actualTable,
```
Comment (satano) [inline Kros.Utils/tests/Kros.Utils.UnitTests/Data/BulkInsert/SqlServerBulkHelper.cs]: Arguments fits on one line.

File: Kros.Utils/tests/Kros.Utils.UnitTests/Data/BulkInsert/SqlServerBulkHelper.cs [hunk 6]
```diff
@@ -61,20 +75,35 @@ private static void CompareColumns(DataTable actualTable, DataTable expectedTabl
...
+                                               DataTable expectedTable,
+                                               DataRow expectedRow)
+        {
+            CompareRowOneColumn(actualTable, expectedTable, expectedRow, string.Empty);
+        }
+
+        public static void CompareRowOneColumn(DataTable actualTable,
```
Comment (satano) [inline Kros.Utils/tests/Kros.Utils.UnitTests/Data/BulkInsert/SqlServerBulkHelper.cs]: Do not use Visual Basic style parameters breaks. If the will not fit into one line do it [this way](https://github.com/Kros-sk/Kros.Libs/blob/master/Kros.Utils/src/Kros.Utils/Data/Schema/ForeignKeySchema.cs#L37-L43).

File: Kros.Utils/tests/Kros.Utils.UnitTests/Data/BulkInsert/SqlServerBulkHelper.cs [hunk 7]
```diff
@@ -90,12 +119,15 @@ private static void CompareRowData(DataRow actualRow, DataRow expectedRow)
 
             foreach (DataColumn column in expectedRow.Table.Columns)
             {
-                object expectedValue = expectedRow[column] == DBNull.Value ? "NULL" : expectedRow[column];
-                object actualValue = actualRow[column.ColumnName] == DBNull.Value ? "NULL" : actualRow[column.ColumnName];
+                if (string.IsNullOrEmpty(columnName) || columnName.ToUpper().Equals(column.ColumnName.ToUpper()))
```
Comment (satano) [inline Kros.Utils/tests/Kros.Utils.UnitTests/Data/BulkInsert/SqlServerBulkHelper.cs]: Use `Equals` with `StringComparison` overload insted of `ToUpper`.

**Pull request merged**